README.md

```diff
@@ -60,13 +60,10 @@ The recommended way to use the KCL for Java is to consume it from Maven.
 
 ## Release Notes
 
-### Latest Release (2.2.0 - April 8, 2019)
-[Milestone#31](https://github.com/awslabs/amazon-kinesis-client/milestone/31)
-* Updated License to [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0)
-  * [PR#523](https://github.com/awslabs/amazon-kinesis-client/pull/523)
-* Introducing configuration for suppressing logs from ReadTimeoutExceptions caused while calling SubscribeToShard.  
-Suppression can be configured by setting `LifecycleConfig#readTimeoutsToIgnoreBeforeWarning(Count)`.
-  * [PR#528](https://github.com/awslabs/amazon-kinesis-client/issues/528)
+### Release 2.2.1 (June 28, 2019)
```

Comment:
Let this be `### Latest Release (2.2.1 - June 28, 2019)`

Reply:
Fixed